### PR TITLE
Remove popup for kids

### LINKED ISSUE #1001: El popup de resultados en el layout de kids es bastante molesto
Sobre todo cuando se lo usa de una tablet o celular. En el mismo se muestra:

* El estado de la solución
* Las expectativas
* Los tableros inicial, final, esperado
* Link a reportar un error y comunidad

Creo que deberíamos pensar una forma de mostrar los resultados sin lanzar un popup.

Primera propuesta:

* Cambiar el personaje según el estado
* Cambiar el color del progress-bar

### PR DESCRIPTION
**Pull request by luchotc**
Resolves #1001 and for transition it solves https://github.com/mumuki/mumuki-gobstones-runner/issues/89

### CODE REVIEW

File: app/assets/javascripts/application/kids.js [hunk 1]
```diff
@@ -91,20 +91,91 @@ mumuki.load(function () {
...
+      $bubble.find('.mu-kids-character-speech-bubble-failed').hide();
+      Object.keys(mumuki.kids.resultAction).forEach($bubble.removeClass.bind($bubble));
+      mumuki.kids.getOverlay().hide();
+    },
+
+    _showFailedMessage: function (data) {
+      var $bubble = mumuki.kids.getCharacterBubble();
+      $bubble.find('.mu-kids-character-speech-bubble-tabs').hide();
+      $bubble.find('.mu-kids-character-speech-bubble-normal').hide();
+      $bubble.find('.mu-kids-character-speech-bubble-failed').show().html(data.title_html);
```
Comment (flbulgarelli) [inline app/assets/javascripts/application/kids.js]: This code is duplicated in last function

File: app/assets/stylesheets/application/modules/_kids.scss [hunk 2]
```diff
@@ -24,22 +25,24 @@ $kids-speech-tabs-width: 40px;
...
     padding-right: 15px;
   }
-  .mu-kids-exercise-board {
-    hr {
-      border-top: $kids-speech-border;
+
+  .mu-kids-states {
+    margin-bottom: 15px;
+    .mu-state-final {
+      // border-top: $kids-speech-border;
```
Comment (flbulgarelli) [inline app/assets/stylesheets/application/modules/_kids.scss]: What about this?

File: app/assets/stylesheets/application/modules/_kids.scss [hunk 3]
```diff
@@ -214,7 +255,7 @@ $kids-speech-tabs-width: 40px;
           border-radius: 12px;
         }
         display: flex;
-        margin-top: 15px;
+        //margin-top: 15px;
```
Comment (flbulgarelli) [inline app/assets/stylesheets/application/modules/_kids.scss]: And here?

File: app/assets/javascripts/application/kids.js [hunk 4]
```diff
@@ -91,20 +91,91 @@ mumuki.load(function () {
...
+      return $('.mu-kids-overlay');
+    },
+
+    showResult: function (data) {  // This function is called by the custom runner
+      mumuki.updateProgressBarAndShowModal(data);
+      if (data.guide_finished_by_solution) return;
+      mumuki.kids.resultAction[data.status](data);
+    },
+
+    restart: function () {  // This function is called by the custom runner
```
Comment (luchotc) [inline app/assets/javascripts/application/kids.js]: https://github.com/mumuki/mumuki-laboratory/issues/982

File: app/assets/javascripts/application/kids.js [hunk 5]
```diff
@@ -91,20 +91,91 @@ mumuki.load(function () {
...
+
+    getSubmissionResult: function () {
+      return $('.submission-results');
+    },
+
+    getOverlay: function () {
+      return $('.mu-kids-overlay');
+    },
+
+    showResult: function (data) {  // This function is called by the custom runner
```
Comment (luchotc) [inline app/assets/javascripts/application/kids.js]: https://github.com/mumuki/mumuki-laboratory/issues/982

**Pull request merged**